src/test/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtEmailServiceTest.java

```diff
@@ -0,0 +1,161 @@
+    private static final String TEST_EMAIL_EXPLANATION2 = "explanation2";
+    private static final String TEST_EMAIL_EXPLANATION3 = "explanation3";
+
+    private static final String TEST_EMAIL_DESCRIPTION = "description";
+    private static final String TEST_EMAIL_DESCRIPTION2 = "description2";
+    private static final String TEST_EMAIL_DESCRIPTION3 = "description3";
+
+    private static final String TEST_EMAIL_DESCRIPTION_CY = "description cy";
+    private static final String TEST_EMAIL_DESCRIPTION_CY2 = "description cy2";
+    private static final String TEST_EMAIL_DESCRIPTION_CY3 = "description cy3";
+
+    private static final String TEST_EMAIL_EXPLANATION_CY = "explanation cy";
+    private static final String TEST_EMAIL_EXPLANATION_CY2 = "explanation cy2";
+    private static final String TEST_EMAIL_EXPLANATION_CY3 = "explanation cy3";
+
+    private static final uk.gov.hmcts.dts.fact.entity.EmailType EMAIL_TYPE1 =
+        new uk.gov.hmcts.dts.fact.entity.EmailType(TEST_EMAIL_TYPE, TEST_EMAIL_DESCRIPTION, TEST_EMAIL_DESCRIPTION_CY);
+    private static final uk.gov.hmcts.dts.fact.entity.EmailType EMAIL_TYPE2 =
+        new uk.gov.hmcts.dts.fact.entity.EmailType(TEST_EMAIL_TYPE2, TEST_EMAIL_DESCRIPTION2, TEST_EMAIL_DESCRIPTION_CY2);
+    private static final uk.gov.hmcts.dts.fact.entity.EmailType EMAIL_TYPE3 =
+        new uk.gov.hmcts.dts.fact.entity.EmailType(TEST_EMAIL_TYPE3, TEST_EMAIL_DESCRIPTION3, TEST_EMAIL_DESCRIPTION_CY3);
+    private static final List<uk.gov.hmcts.dts.fact.entity.EmailType> EMAIL_TYPES =
+        Arrays.asList(EMAIL_TYPE1, EMAIL_TYPE2, EMAIL_TYPE3);
+
+    private static final int EMAIL_COUNT = 3;
+
+    private static final List<CourtEmail> COURT_EMAILS = new ArrayList<>();
+
+    private static final List<Email> EXPECTED_EMAILS = Arrays.asList(
+        new Email(TEST_EMAIL_ADDRESS, TEST_EMAIL_EXPLANATION, TEST_EMAIL_EXPLANATION_CY, TEST_EMAIL_TYPE),
+        new Email(TEST_EMAIL_ADDRESS2, TEST_EMAIL_EXPLANATION2, TEST_EMAIL_EXPLANATION_CY2, TEST_EMAIL_TYPE2),
+        new Email(TEST_EMAIL_ADDRESS3, TEST_EMAIL_EXPLANATION3, TEST_EMAIL_EXPLANATION_CY3, TEST_EMAIL_TYPE3)
+    );
+
+    @Autowired
+    private AdminCourtEmailService adminService;
+
+    @MockBean
+    private CourtRepository courtRepository;
+
+    @MockBean
+    private CourtEmailRepository emailRepository;
+
+    @MockBean
+    private EmailTypeRepository emailTypeRepository;
+
+    @Mock
+    private Court court;
+
+    @BeforeAll
+    static void setUp() {
+
+        for (int i = 0; i < EMAIL_COUNT; i++) {
+            CourtEmail courtEmail = mock(CourtEmail.class);
+            COURT_EMAILS.add(courtEmail);
+        }
+
+        when(COURT_EMAILS.get(0).getEmail())
+            .thenReturn(new uk.gov.hmcts.dts.fact.entity.Email(
+                TEST_EMAIL_ADDRESS, TEST_EMAIL_EXPLANATION, TEST_EMAIL_EXPLANATION_CY,
+                new uk.gov.hmcts.dts.fact.entity.EmailType(TEST_EMAIL_TYPE,
```

Comment:
Is this not the same as EMAIL_TYPE1 above. If so we can just reuse it here

Reply:
Different values, just to check it isn't assigning the value of like element 1 to both element 2 and 3 etc :) 

If that's not a concern, can use the same values for all three